lib/shortcode/easy-digital-downloads.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+/**
+ * Shortcode Template File for Easy Digital Downloads
+ */
+
+
+/**
+ * Create our shortcode for Easy Digital Downloads
+ */
+add_action( 'init', 'sandwich_easy_digital_downloads', 11 );
+function sandwich_easy_digital_downloads() {
+
+	// Check if Shortcake exists
+	if ( ! function_exists( 'shortcode_ui_register_for_shortcode' ) ) {
+		return;
+	}
+
+	// Register Shortcake UI for Easy Digital Downloads Download History
+	shortcode_ui_register_for_shortcode( 'download_history', 
+		array(
+			'label' => __( 'Easy Digital Downloads - Download History', 'pbsandwich' ),
+			'listItemImage' => 'dashicons-wordpress download-count',
+		) 
+	);
+	
+	// Register Shortcake UI for Easy Digital Downloads Receipt
+	shortcode_ui_register_for_shortcode(
+		'edd_receipt',
+		array(
+			'label' => __( 'Easy Digital Downloads - Receipt', 'pbsandwich' ),
+			'listItemImage' => 'dashicons-wordpress download-count',
+			'attrs' => array(
+				array(
+					'label' => __( 'Error Message', 'pbsandwich' ),
```

Comment:
Does EDD have translation text for these? Because if they have, we can use that to keep our `pbsandwich` translations to the minimum